python/sdk/tests/test_detect_controller.py

```diff
@@ -101,12 +101,6 @@ def test_disable_test(self, unwrap):
         queue = unwrap(self.iam.get_account)(self.iam)['queue']
         assert len([test for test in queue if test['test'] == pytest.test_id]) == 0
 
-    @pytest.mark.order(after='test_describe_activity')
-    def test_social_stats(self, unwrap):
-        """Test social_stats method"""
-        res = unwrap(self.detect.social_stats)(self.detect, ident=self.common_ransomware)
-        assert len(res.values()) >= 1
-
     @pytest.mark.order(after='test_social_stats')
```

Comment:
Need to adjust this test so that it follows the `test_describe_activity` action now

Reply:
ah thx

Reply:
@stephanwampouille assist?

Reply:
fixed

Reply:
thanks!